themes/default/content/blog/developer-experience-infrastructure-kubernetes-automation-with-okteto-pulumi/index.md

```diff
@@ -0,0 +1,91 @@
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-03-06T14:37:09Z
+
+# The draft setting determines whether a post is published. Set it to true if
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: "Platform engineers need to prioritize automating cloud resource provisioning. Watch the recording of this DevOps & Platform workshop to learn how to do it with Pulumi and Okteto."
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - arsh-sharma
+    - sara-huddleston
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - developer-experience-devex
+    - devops
+    - platform-engineering
+    - kubernetes
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+In today's landscape, it is common to build microservices-based applications that leverage resources like RDS databases, storage buckets, and more, from various cloud platforms. Consequently, providing developers access to these resources is crucial during the development phase. Failing to do so forces developers to code in an unrealistic environment, resulting in difficulties and the introduction of bugs. To tackle this challenge, platform engineers should prioritize automating the process. This article will explore how Pulumi and Okteto can effortlessly facilitate this automation.
+
+<!--more-->
+
+Pulumi and Okteto seamlessly integrate to automate the setup of development infrastructure for microservices-based applications. Leveraging Okteto's "External Resources" feature, you can employ your existing Pulumi configuration designed for production to create the development infrastructure seamlessly. What the workshop with a hands-on demo below.
+
+***This article is based on Automate Provisioning Any Dev Resource on Any Cloud Provider With Pulumi and Okteto in Okteto's blog.***
```

Comment:
I'd recommend linking back to the Okteto blog post here.

Reply:
I have it linked after the workshop video... or is there a way that I can link and force it to open a new tab instead of the same?